src/pages/Dashboards/Nav/NavItem/NavItem.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import cx from 'classnames'
+import Svg from 'webkit/ui/Svg/react'
+import styles from './NavItem.module.scss'
+
+const SubItem = ({ subItem, activeSub, scrollToSubItem }) => {
+  const { key, title } = subItem
+  const isActiveSub = key === activeSub.key
+
+  function handleSubItemClick() {
+    scrollToSubItem(subItem)
+  }
+
+  return (
+    <li className={cx(styles.subItemWrapper, isActiveSub && styles.activeSub, 'mrg-xs mrg--b')}>
+      <button className={cx(styles.subItem, 'btn')} onClick={handleSubItemClick}>
+        {title}
+      </button>
+    </li>
+  )
+}
+
+const NavItem = ({ isActive, activeSub, setActive, scrollToSubItem, ...item }) => {
+  const { title, subItems } = item
+
+  let subNav = null
+
+  if (activeSub) {
+    subNav = (
+      <ul className={cx(styles.subWrapper)}>
+        {subItems.map((subItem) => (
+          <SubItem
+            key={subItem.key}
+            activeSub={activeSub}
+            subItem={subItem}
+            scrollToSubItem={scrollToSubItem}
+          />
+        ))}
+      </ul>
+    )
+  }
```

Comment:
Conditionally render element on the `LOC 56` -> `{activeSub && <ul className ....}`